cmd/flipt/export.go

```diff
@@ -51,10 +52,17 @@ func newExportCommand() *cobra.Command {
 	)
 
 	cmd.Flags().StringVarP(
-		&export.namespace,
-		"namespace", "n",
+		&export.namespaces,
+		"namespaces", "n",
 		flipt.DefaultNamespace,
-		"source namespace for exported resources.",
+		"comma delimited list of namespaces to export from.",
+	)
+
+	cmd.Flags().BoolVar(
+		&export.allNamespaces,
+		"all-namespaces",
+		false,
+		"boolean flag to specify if every namespace should be exported.",
```

Comment:
```suggestion
		"export all namespaces",
```

Reply:
we should probably add a check so that `all-namespaces` and `namespaces` are mutually exclusive

https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive

Reply:
@markphelps Yup, can do this.